docs/zh_cn/04-developer-guide/do_regression_test.md.md

```diff
@@ -49,56 +68,54 @@ python ./tools/regression_test.py \
 
 ## 例子
 
-1. 测试 mmdet 和 mmpose 的所有 backend 的 转换+精度
+1. 测试 mmdet 和 mmpose 的所有 backend 的 **转换+精度**
 
 ```shell
 python ./tools/regression_test.py \
-    --deploy-yml ./configs/mmdet/mmdet_regression_test.yaml ./configs/mmpose/mmpose_regression_test.yaml \
+    --codebase mmdet mmpose \
     --backends all \
     --work-dir "../mmdeploy_regression_working_dir" \
     --device "cuda" \
     --log-level INFO \
     --performance
 ```
 
-2. 测试 mmdet 和 mmpose 的某几个 backend 的 转换+精度
+2. 测试 mmdet 和 mmpose 的某几个 backend 的 **转换+精度**
 
 ```shell
 python ./tools/regression_test.py \
-    --deploy-yml ./configs/mmdet/mmdet_regression_test.yaml ./configs/mmdet/mmpose.yaml \
+    --codebase mmdet mmpose \
     --backends onnxruntime tesnsorrt \
     --work-dir "../mmdeploy_regression_working_dir" \
     --device "cuda" \
     --log-level INFO \
-    --performance
+    -p
 ```
 
-3. 测试 mmdet 和 mmpose 的某几个 backend，只需测试转换
+3. 测试 mmdet 和 mmpose 的某几个 backend，**只测试转换**
 
 ```shell
 python ./tools/regression_test.py \
-    --deploy-yml ./configs/mmdet/mmdet_regression_test.yaml ./configs/mmdet/mmpose.yaml \
+    --codebase mmdet mmpose \
     --backends onnxruntime tesnsorrt \
     --work-dir "../mmdeploy_regression_working_dir" \
     --device "cuda" \
```

Comment:
openvino needs device=cpu. ort could be device=cpu/cuda and it depends whethere onnxruntime-gpu is installed. should deal with this special case inside regression_test.py

Reply:
Done

Reply:
The script will switch to cpu when testing openvino and onnxruntime(cpu version)